WordPress/src/main/res/layout/modal_layout_picker_bottom_toolbar.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="center"
+    android:orientation="horizontal"
+    android:paddingBottom="@dimen/mlp_bottom_button_vertical_margin"
+    android:paddingEnd="@dimen/mlp_bottom_button_horizontal_margin"
+    android:paddingStart="@dimen/mlp_bottom_button_horizontal_margin"
+    android:paddingTop="@dimen/mlp_bottom_button_vertical_margin">
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/createBlankPageButton"
+        style="@style/Widget.ModalLayoutPicker.Button.Secondary"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="1"
+        android:text="@string/mlp_create_blank_page" />
+
+    <com.google.android.material.button.MaterialButton
```

Comment:
The two last buttons are not actually needed now. Just added them for future use and to check the layout behavior